changelog.d/20240109_111904_fateme.khodayari97_update_readme_of_custom_mfe_plugin_url.md

```diff
@@ -0,0 +1,4 @@
+- [Improvement] Add warning to `Adding new MFEs` section of README about the URL of custom MFEs. 
```

Comment:
There is no need to add a changelog entry, as this is a change that does not directly affect users.

Reply:
Thanks for mentioning. I'll remove it then.